tests/components/zwave_js/test_climate.py

```diff
@@ -335,3 +336,81 @@ async def test_thermostat_different_endpoints(
     state = hass.states.get(CLIMATE_RADIO_THERMOSTAT_ENTITY)
 
     assert state.attributes[ATTR_CURRENT_TEMPERATURE] == 22.5
+
+
+async def test_setpoint_thermostat(hass, client, climate_danfoss_lc_13, integration):
+    """Test a setpoint thermostat command class entity."""
+    node = climate_danfoss_lc_13
+    state = hass.states.get(CLIMATE_DANFOSS_LC13_ENTITY)
+
+    assert state
+    assert state.state == HVAC_MODE_HEAT
+    assert state.attributes[ATTR_TEMPERATURE] == 25
+    assert state.attributes[ATTR_HVAC_MODES] == []
```

Comment:
This should include the current mode. 

```suggestion
    assert state.attributes[ATTR_HVAC_MODES] == [HVAC_MODE_HEAT]
```